### PR TITLE
feat(staking): update validator

### PR DESCRIPTION
**Pull request by Ramarti**
Updates validator commission method, refactors `unjailFee` into `validatorUpdateFee`, charges it fee in methods updating a validator's state

issue: none

### CODE REVIEW

**Review by jdubpark — APPROVED** [review]

File: contracts/src/interfaces/IIPTokenStaking.sol [hunk 1]
```diff
@@ -23,7 +23,7 @@ interface IIPTokenStaking {
     /// @param shortStakingPeriod The staking duration for short staking, in seconds
     /// @param mediumStakingPeriod The staking duration for medium staking, in seconds
     /// @param longStakingPeriod The staking duration for long staking, in seconds
-    /// @param unjailFee The fee required to unjail a validator
+    /// @param validatorUpdateFee The fee required to unjail a validator
```
Comment (0xHansLee) [inline contracts/src/interfaces/IIPTokenStaking.sol:26]: Is this the fee that required to update validator info? including unjail. If so, let's update comment as well.

File: contracts/src/protocol/IPTokenStaking.sol [hunk 2]
```diff
@@ -31,7 +31,7 @@ contract IPTokenStaking is IIPTokenStaking, Ownable2StepUpgradeable, ReentrancyG
     /// @notice Stake amount increments, 1 ether => e.g. 1 ether, 2 ether, 5 ether etc.
     uint256 public immutable STAKE_ROUNDING;
 
-    /// @notice Default minimum unjail fee
+    /// @notice Default minimum validator update fee
     uint256 public immutable DEFAULT_MIN_UNJAIL_FEE;
```
Comment (0xHansLee) [inline contracts/src/protocol/IPTokenStaking.sol:35]: need to be changed as well?

File: contracts/src/protocol/IPTokenStaking.sol [hunk 3]
```diff
@@ -79,15 +79,23 @@ contract IPTokenStaking is IIPTokenStaking, Ownable2StepUpgradeable, ReentrancyG
...
             revert Errors.IPTokenStaking__ZeroStakingRounding();
         }
         STAKE_ROUNDING = stakingRounding; // Recommended: 1 gwei (10^9)
-        if (defaultMinUnjailFee < 1 gwei) {
-            revert Errors.IPTokenStaking__InvalidDefaultMinUnjailFee();
+        if (defaultMinValidatorUpdateFee < 1 gwei) {
+            revert Errors.IPTokenStaking__InvalidDefaultMinValidatorUpdateFee();
         }
-        DEFAULT_MIN_UNJAIL_FEE = defaultMinUnjailFee;
+        DEFAULT_MIN_UNJAIL_FEE = defaultMinValidatorUpdateFee;
```
Comment (edisonz0718) [inline contracts/src/protocol/IPTokenStaking.sol:98]: Need to change the name for this? It's still unjail

**Review by limengformal — APPROVED** [review]

File: contracts/src/interfaces/IIPTokenStaking.sol (hunk 1 above)
Comment (Ramarti) [reply contracts/src/interfaces/IIPTokenStaking.sol:26]: Refactoring to "fee" so it applies to operator methods as well

File: contracts/src/protocol/IPTokenStaking.sol (hunk 2 above)
Comment (Ramarti) [reply contracts/src/protocol/IPTokenStaking.sol:35]: Refactoring to "fee" so it applies to operator methods as well

**Review by 0xHansLee — APPROVED** [review]
LGTM

**Pull request merged**